inference-engine/tests/functional/plugin/shared/include/behavior/ov_exec_network.hpp

```diff
@@ -51,21 +166,217 @@ TEST_P(OVExecNetwork, precisionsAsInOriginalIR) {
+    ASSERT_NO_THROW(importedExecNet.output("round"));
+}
+
+TEST_P(OVExecNetwork, importExportedIENetwork) {
+    // Skip test according to plugin specific disabledTestPatterns() (if any)
+    SKIP_IF_CURRENT_TEST_IS_DISABLED();
+    InferenceEngine::Core core;
+    InferenceEngine::ExecutableNetwork execNet;
+
+    // Create simple function
+    {
+        auto param1 = std::make_shared<ov::opset8::Parameter>(elementType, ngraph::Shape({1, 3, 24, 24}));
+        param1->set_friendly_name("param1");
+        param1->output(0).get_tensor().set_names({"data1"});
+        auto param2 = std::make_shared<ov::opset8::Parameter>(elementType, ngraph::Shape({1, 3, 24, 24}));
+        param2->set_friendly_name("param2");
+        param2->output(0).get_tensor().set_names({"data2"});
+        auto relu = std::make_shared<ov::opset8::Relu>(param1);
+        relu->set_friendly_name("relu_op");
+        relu->output(0).get_tensor().set_names({"relu"});
+        auto result1 = std::make_shared<ov::opset8::Result>(relu);
+        result1->set_friendly_name("result1");
+        auto concat = std::make_shared<ov::opset8::Concat>(OutputVector{relu, param2}, 1);
+        concat->set_friendly_name("concat_op");
+        concat->output(0).get_tensor().set_names({"concat"});
+        auto result2 = std::make_shared<ov::opset8::Result>(concat);
+        result2->set_friendly_name("result2");
+        function = std::make_shared<ngraph::Function>(ngraph::ResultVector{result1, result2},
+                                                      ngraph::ParameterVector{param1, param2});
+        function->set_friendly_name("SingleRuLU");
+    }
+    ASSERT_NO_THROW(execNet = core.LoadNetwork(InferenceEngine::CNNNetwork(function), targetDevice, configuration));
+
+    std::stringstream strm;
+    execNet.Export(strm);
+
+    ov::runtime::ExecutableNetwork importedExecNet = ie->import_model(strm, targetDevice, configuration);
+    ASSERT_EQ(function->inputs().size(), 2);
+    ASSERT_EQ(function->inputs().size(), importedExecNet.inputs().size());
+    ASSERT_THROW(importedExecNet.input(), ov::Exception);
+    ASSERT_THROW(importedExecNet.input("data1").get_node(), ov::Exception);
+    ASSERT_THROW(importedExecNet.input("data2").get_node(), ov::Exception);
+    ASSERT_NO_THROW(importedExecNet.input("param1").get_node());
+    ASSERT_NO_THROW(importedExecNet.input("param2").get_node());
+    ASSERT_EQ(function->outputs().size(), 2);
+    ASSERT_EQ(function->outputs().size(), importedExecNet.outputs().size());
+    ASSERT_THROW(importedExecNet.output(), ov::Exception);
+    ASSERT_NE(function->output(0).get_tensor().get_names(), importedExecNet.output(0).get_tensor().get_names());
+    ASSERT_THROW(importedExecNet.output("relu").get_node(), ov::Exception);
+    ASSERT_THROW(importedExecNet.output("concat").get_node(), ov::Exception);
+    ASSERT_NO_THROW(importedExecNet.output("relu_op").get_node());
+    ASSERT_NO_THROW(importedExecNet.output("concat_op").get_node());
```

Comment:
can we also check that precision are imported correctly? i.e. with CNNNetwork's conversion is applied
but need to change original precisions to f16